public-ui/src/app/data-browser/components/survey-answer/survey-answer-react.component.tsx

```diff
@@ -0,0 +1,255 @@
+});
+
+interface Props {
+    isCopeSurvey: boolean;
+    question: any;
+    particpantCount: any;
+}
+
+export const SurveyAnswerReactComponent = (class extends React.Component<Props, {}> {
+    constructor(props: Props) {
+        super(props);
+    }
+
+    isSubTable = this.props.particpantCount ? true : false;
+
+    render(): any {
+        return <React.Fragment><style>{styleCss}</style>
+            <div className='survey-tbl'>
+                <div className='survey-tbl-r survey-tbl-head'>
+                    <div className='info-text first survey-tbl-d'>
+                        Answer
+                    </div>
+                    <div className='survey-tbl-r-group survey-tbl-r-group-style '>
+                        <div className='info-text survey-tbl-d display-body'>
+                            {this.props.isCopeSurvey ? <h1>testIs cope</h1> : <span><React.Fragment>Concept Code</React.Fragment>
+                                <TooltipReactComponent tooltipKey='conceptCodeHelpText' label='test' searchTerm='test' action='Survey Page Tooltip' /></span>
+                            }
+                        </div >
+                        <div className='info-text survey-tbl-d display-body'>
+                            {this.props.isCopeSurvey ? <h1>testIs cope</h1> : <span><React.Fragment>Participant Count</React.Fragment>
+                                <TooltipReactComponent tooltipKey='surveyParticipantCountHelpText' label='test' searchTerm='test' action='Survey Page Tooltip' /></span>
+                            }
+                        </div >
+                        <div className='info-text survey-tbl-d display-body'>
+                            {this.props.isCopeSurvey ? <h1>testIs cope</h1> :
+                                <span>{this.isSubTable ? <React.Fragment>% Answered out of {this.props.particpantCount}</React.Fragment> :
+                                    <React.Fragment>% Answered </React.Fragment>}</span>
+                            }
+                        </div >
+                        <div className='info-text survey-tbl-d display-body'>
+                            {this.props.isCopeSurvey ? <h1>testIs cope</h1> : <React.Fragment></React.Fragment>
+                            }
+                        </div >
+                    </div >
+                </div >
+
+                {
+                   this.props.question.countAnalysis ? 
+                    this.props.question.countAnalysis.results.map((answer, index) => {
+
+                        if (answer.stratum4 !== 'Did not answer') {
+                            const key = 'answer' + index;
+                            return <SurveyAnswerRowComponent partcipantCount={this.props.particpantCount}
+                                key={key}
+                                isCopeSurvey={this.props.isCopeSurvey}{...answer} />;
+                        }
+                    }) : undefined
+                
+                } 
```

Comment:
Should use `&&` instead of ternary operator (may need `!!` here as well if `countAnalysis` is not a boolean:
```suggestion
                {!!this.props.question.countAnalysis && this.props.question.countAnalysis.results.map((answer, index) => {
                    if (answer.stratum4 !== 'Did not answer') {
                        const key = 'answer' + index;
                        return <SurveyAnswerRowComponent partcipantCount={this.props.particpantCount}
                            key={key}
                            isCopeSurvey={this.props.isCopeSurvey}{...answer} />;
                    }
                })}
```